samples/unity-of-bugs/Assets/Plugins/Sentry/SentryCliOptions.asset

```diff
@@ -0,0 +1,19 @@
+%YAML 1.1
```

Comment:
It looks like I forgot to add SentryCliOptions to .gitignore

Reply:
removed & added to gitignore